includes/service-providers/class-newspack-newsletters-service-provider.php

```diff
@@ -336,6 +336,24 @@ public function insert_post_data( $data, $postarr ) {
 		return $data;
 	}
 
+	/**
+	 * Notify the site administrators.
+	 *
+	 * @param string $subject The email subject.
+	 * @param string $message The email message.
+	 */
+	private static function notify_site_admins( $subject, $message ) {
+		$site_administrators = get_users(
+			[
+				'role'  => 'administrator',
+				'limit' => 10, // Notifying 10 people should be enough.
+			]
+		);
+		foreach ( $site_administrators as $user ) {
+			$sent = \wp_mail( $user->user_email, $subject, $message ); // phpcs:ignore WordPressVIPMinimum.Functions.RestrictedFunctions.wp_mail_wp_mail
```

Comment:
This is the email I receive when my newsletter fails:

![Screenshot 2024-02-27 at 17 03 37](https://github.com/Automattic/newspack-newsletters/assets/17905991/480a928e-7aa2-4ac1-b611-684ed98d61a8)

It feels a bit bare and doesn't really provide me with too much information. We might want to include some additional details like:

- Some details about which newsletter failed
- related to the first point, maybe a link to their newsletter or newsletter settings in wp-admin
- Maybe even a support link if they can't figure out how to address this on their end

Reply:
Updated in cfa11e6255df7be8427ea4af6e44b11a3bde6ab0 to include campaign title and edit link.